README.md

```diff
@@ -47,6 +47,9 @@ $ cd $GOPATH/src/github.com/example-inc/
 $ operator-sdk new app-operator --api-version=app.example.com/v1alpha1 --kind=App
 $ cd app-operator
 
+# Test the app-operator locally by setting KUBERNETES_CONFIG
+$ KUBERNETES_CONFIG=~/.kube/config go run cmd/app-operator/main.go
```

Comment:
Let's not add this `KUBERNETES_CONFIG=~/.kube/config go run cmd/app-operator/main.go` to the readme yet.

I want to wrap the above in a new `operator-sdk up` command. I will add that in a separate pr.